activation/licensed-users-count.md

```diff
@@ -0,0 +1,32 @@
+---
+description: This article explains how Microsoft 365 Licences work with SysKit Point.
```

Comment:
prijedlog: This article explains how SysKit Point licensing depends on Microsoft 365 licenses in your tenant.